### PR TITLE
queries for deliverables 6-11

### PR DESCRIPTION
**Pull request by galenhwang**
All functions for the queries have been tested on the non-UI endpoints and return json objects as desired; will have to be called by UI endpoints pending a merge.

### CODE REVIEW

**Comment by jefftwin** [comment]
there are merge conflicts, please fix the merge conflicts so we get to review the final code

File: airplanedb.py [hunk 1]
```diff
@@ -1423,10 +1493,6 @@ def get_schedule_for_employee(self, employee_id):
             db.rollback()
             data = 0
```
Comment (maknoon) [inline airplanedb.py:1512]: why did `return data` get removed here?

File: airplanedb.py [hunk 2]
```diff
@@ -1423,10 +1493,6 @@ def get_schedule_for_employee(self, employee_id):
...
 
-        cursor.close()
```
Comment (jefftwin) [inline airplanedb.py:1443]: This stuff shouldn't be deleted

**Review by jefftwin — CHANGES_REQUESTED** [review]

File: airplanedb.py (hunk 1 above)
Comment (jefftwin) [reply airplanedb.py:1512]: oh lel, caught the same thing

File: airplanedb.py [hunk 3]
```diff
@@ -1701,38 +1705,6 @@ def get_flight(self, f_id):
         db.close()
         return data
 
-#==============================================================================
```
Comment (maknoon) [inline airplanedb.py]: why is this getting removed? is it being added somewhere else?

**Review by jefftwin — APPROVED** [review]

**Pull request merged**